### PR TITLE
Fix windows resize a11y

### PR DESCRIPTION
**Pull request by bsclifton**
Includes fixes I needed to get compiling on Windows (may or may not be merged already; fixes + lint fix)

I sent a copy of the binary to @jonathansampson via Slack; please try it out and let me know how it looks (with and without the option enabled)

### CODE REVIEW

File: atom/browser/native_window_views_win.cc [hunk 1]
```diff
@@ -72,6 +72,12 @@ const char* AppCommandToString(int command_id) {
   }
 }
 
+bool IsScreenReaderActive() {
```
Comment (bsclifton) [inline atom/browser/native_window_views_win.cc]: This block courtesy of https://github.com/electron/electron/commit/9c9ba809783b838cd1ff06c974bb47e2cfc40e9d

**Comment by bridiver** [comment]
please rebase and I will merge

File: brave/common/extensions/asar_source_map.cc [hunk 2]
```diff
@@ -27,7 +27,7 @@ bool ReadFromPath(
     const base::FilePath& path,
     std::string* source) {
   base::FilePath file_path = path.Append(file);
-  if (!file_path.MatchesExtension(".js"))
+  if (!file_path.MatchesExtension(FILE_PATH_LITERAL(".js")))
     file_path = file_path.AddExtension(FILE_PATH_LITERAL("js"));
```
Comment (bsclifton) [inline brave/common/extensions/asar_source_map.cc]: This is needed or the compile fails (versus the raw ".js" comparison)

**Comment by bsclifton** [comment]
@bridiver rebased 😄

File: brave/common/extensions/asar_source_map.cc (hunk 2 above)
Comment (bsclifton) [inline brave/common/extensions/asar_source_map.cc]: Specific compile error (why I needed to add the FILE_PATH_LITERAL wrapper):
```
FAILED: obj/electron/common/asar_source_map.obj
ninja -t msvc -e environment.x64 -- "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64/cl.exe" /nologo /showIncludes /FC @obj/electron/common/asar_source_map.obj.rsp /c ../../electron/brave/common/extensions/asar_source_map.cc /Foobj/electron/common/asar_source_map.obj /Fd"obj/electron/common_cc.pdb"
c:\github\browser-laptop-bootstrap\src\electron\brave\common\extensions\asar_source_map.cc(28): error C2664: 'bool base::FilePath::MatchesExtension(base::FilePath::StringPieceType) const': cannot convert argument 1 from 'const char [4]' to 'base::FilePath::StringPieceType'
c:\github\browser-laptop-bootstrap\src\electron\brave\common\extensions\asar_source_map.cc(28): note: No constructor could take the source type, or constructor overload resolution was ambiguous
```

**Comment by bsclifton** [comment]
I moved `libs = [ "uiautomationcore.lib" ]` over to `atom/browser/BUILD.gn` per feedback (see https://github.com/brave/muon/pull/169/commits/fa222b6fe23684bd1d4d756ee5b2ba19666d6241)

**Comment by bsclifton** [comment]
If needed, https://github.com/brave/muon/pull/169/commits/749324ffc318f30fc1215ba016afbd4ea3123180 can be cherry-picked (I'm testing it out on macOS right now)

**Pull request merged**